src/material/chips/testing/chip-harness-filters.ts

```diff
@@ -7,21 +7,30 @@
  */
 import {BaseHarnessFilters} from '@angular/cdk/testing';
 
-/** A set of criteria that can be used to filter a list of `MatChipHarness` instances. */
+/** A set of criteria that can be used to filter a list of chip instances. */
 export interface ChipHarnessFilters extends BaseHarnessFilters {
   /** Only find instances whose text matches the given value. */
   text?: string | RegExp;
   /**
    * Only find chip instances whose selected state matches the given value.
-   * @deprecated Will be moved into separate selection-specific harness.
+   * @deprecated Use `MatSelectableChipHarness` together with `SelectableChipHarnessFilters`.
    * @breaking-change 12.0.0
    */
   selected?: boolean;
 }
 
-/** A set of criteria that can be used to filter a list of `MatChipListHarness` instances. */
+/** A set of criteria that can be used to filter a list of selectable chip instances. */
+export interface SelectableChipHarnessFilters extends ChipHarnessFilters {
```

Comment:
Could we name these like `ChipOptionHarnessFilters` and `ChipListboxHarnessFilters`? While "selectable" is accurate, I want to move more in the direction of naming things around the underlying aria pattern (aligning with the MDC API).

Similar for the actual harness classes/files themselves.